docs/contributing-to-airbyte/writing-docs.md

```diff
@@ -84,6 +84,72 @@ To make complex changes or edit multiple files, edit the files on your local mac
 
 5. Assign `airbytehq/docs` as a Reviewer for your pull request. 
 
+### Custom markdown extensions for connector docs
+Airbyte's markdown documentation—particularly connector-specific documentation—needs to
+gracefully support multiple different contexts: key details may differ between open-source builds
+and Airbyte Cloud, and the more exhaustive explanations appropriate for
+https://docs.airbyte.com may bury key details when rendered as inline documentation within
+the Airbyte application. In order to support all these different contexts
+without resorting to multiple overlapping files that must be maintained in parallel,
+Airbyte's documentation tooling supports multiple nonstandard features. Please familiarize
```

Comment:
```suggestion
Airbyte's documentation tooling supports multiple nonstandard features.

Please familiarize
```
Add a line break here to help improve readability

Reply:
Also, is there a specific reason why you put each line of text on a separate line of markdown here? If not, we should just using single markdown lines for each paragraph to match the approach used in the rest of the file, and to make editing easier.

Reply:
> is there a specific reason why you put each line of text on a separate line of markdown here?

Yes, but not one that justifies leaving it that way—it's easier to navigate multiple lines than a single wrapped line in old-fashioned neckbeard text editors, and the resulting html is identical. Will update, no problem.